### PR TITLE
docs(quickstart): add stonedb-8.0 compiling guide #1449

### LINKED ISSUE #1449: docs: add stonedb-8.0 compiling and running guide
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7-dev/CONTRIBUTING.md).

### Description

Describe the new method to compile and run stonedb-8.0 in developer guide.

### Self-service

- [X] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by Double0101**
## Summary about this PR

Issue Number: close #1449

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Check if  `git` has been installed. If not, install it first.
 1. Issue report: Open a regular [StoneDB issue](https://github.com/stoneatom/stonedb/issues/new/choose) to bind your pull request.<br />For more details about creating github issue concepts, see: [create an github issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue)
 2.  Fork [StoneDB repo](https://github.com/stoneatom/stonedb) to your own account. 
 3.  After add you own code, add unit test 
-4.  Do *make mtr test*
+4.  Do *make mtr test* [How to add mtr test](./Docs/02-getting-started/quick-add-mtr-test.md)
```
Comment (Nliver) [inline CONTRIBUTING.md]: This link should be replaced with an absolute link to the site. Relative files in the Github repository are not recommended, as the files in the Docs directory are for rendering the site.

File: Docs/02-getting-started/quick-add-mtr-test.md
Comment (Nliver) [inline Docs/02-getting-started/quick-add-mtr-test.md:1]: StoneDB except English documents, as well as Chinese documents, corresponding Chinese document directory in this article: `stonedb/website/i18n/useful/docusaurus-plugin-content-docs/current
/02-getting-started/`

File: Docs/04-developer-guide/00-compiling-methods/compile-using-ubuntu2204/compile-using-ubuntu2204-for-80.md
Comment (Nliver) [inline Docs/04-developer-guide/00-compiling-methods/compile-using-ubuntu2204/compile-using-ubuntu2204-for-80.md:1]: StoneDB except English documents, as well as Chinese documents, corresponding Chinese document directory in this article: `stonedb/website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods
/compile-using-ubuntu2004/`

File: Docs/04-developer-guide/00-compiling-methods/compile-using-ubuntu2204/compile-using-ubuntu2204-for-80.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
+---
+id: compile-using-ubuntu22.04-for-80
+sidebar_position: 5.151
```
Comment (Nliver) [inline Docs/04-developer-guide/00-compiling-methods/compile-using-ubuntu2204/compile-using-ubuntu2204-for-80.md:3]: The `5.15` code here is already used in the `stonedb/Docs/04-developer-guide/00-compiling-methods/compile-using-docker.md file`. This file is compiled under Ubuntu. It is recommended that you change the number to `5.140`. 

Of course, the home directory for this number is `using-ubuntu2004`, so it's best if your build method is compatible with `ubuntu20.04` as well

**Review by Nliver — CHANGES_REQUESTED** [review]

**Comment by Nliver** [comment]
As you know, it takes patience to update the documentation, so I have suggested some detailed changes in hopes of helping you understand StoneDB's documentation system, as well as suggesting that you update the corresponding Chinese documentation.

File: website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods/compile-using-ubuntu2204/compile-using-ubuntu2204-for-80.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+sudo apt install -y zstd
+sudo apt install -y lz4
+sudo apt install -y ncurses-dev
+sudo apt install -y libsnappy-dev
+sudo apt install -y libedit-dev
+sudo apt install -y libaio-dev
+sudo apt install -y libncurses5-dev 
+sudo apt install -y libreadline-dev
+sudo apt install -y libpam0g-dev
+sudo apt install -y zlib1g-dev
```
Comment (chenshengjiang) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods/compile-using-ubuntu2204/compile-using-ubuntu2204-for-80.md:53]: 重复

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ Check if  `git` has been installed. If not, install it first.
 1. Issue report: Open a regular [StoneDB issue](https://github.com/stoneatom/stonedb/issues/new/choose) to bind your pull request.<br />For more details about creating github issue concepts, see: [create an github issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue)
 2.  Fork [StoneDB repo](https://github.com/stoneatom/stonedb) to your own account. 
 3.  After add you own code, add unit test 
-4.  Do *make mtr test*
+4.  Do *make mtr test* [How to add mtr test](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-ubuntu2204/compile-using-ubuntu22.04-for-80/)
```
Comment (Nliver) [inline CONTRIBUTING.md]: the link here does not point to the relevant documentation for MTR？

Comment (hustjieke) [inline CONTRIBUTING.md]: not mtr links?

**Review by hustjieke — APPROVED** [review]
LGTM

**Comment by hustjieke** [comment]
@Nliver @haitaoguan 
Please take a CR.

File: Docs/02-getting-started/quick-add-mtr-test.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
+---
+id: quick-add-mtr-test
+sidebar_position: 3.6
+---
+
+# Add and Run a MTR test
```
Comment (Nliver) [inline Docs/02-getting-started/quick-add-mtr-test.md]: The first header is used to render the page title. The following code is recommended instead：
```
# Qucik Add MTR Test

## Add and Run a MTR test

### ......
```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/02-getting-started/quick-add-mtr-test.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
+---
+id: quick-add-mtr-test
+sidebar_position: 3.6
+---
+
+# 添加并运行一个mtr测试用例
```
Comment (Nliver) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/02-getting-started/quick-add-mtr-test.md]: The first header is used to render the page title. The following code is recommended instead：
```
# 快速添加 MTR Test

## 添加并运行一个 mtr 测试用例

### ......
```

**Review by Nliver — CHANGES_REQUESTED** [review]

**Review by Nliver — APPROVED** [review]
LGTM, Good job!

**Review by haitaoguan — APPROVED** [review]

**Pull request merged**